LICENSE.txt

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015 Microsoft
+Copyright (c) Microsoft
```

Comment:
why are you changing the LICENSE.txt at all as part of this change?  I would just revert this file entirely and not up date it.

Reply:
Revert back